spaceapps-orrery/src/components/Orrey/orrey_scene.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect } from 'react';
+import {
+  Scene,
+  WebGLRenderer,
+  PerspectiveCamera,
+} from 'three';
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls';
+import Starfield from './starField'; 
+import Sun from './sun';
+import Planet from './planets';
+
+const planets = [
```

Comment:
The name of each planet so when comparing with the API its easier to associate the static data (the commited) with the dynamic extracted from the API (the mateos commit)